src/NServiceBus.NHibernate/NServiceBus.NHibernate.csproj

```diff
@@ -1,32 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net452</TargetFrameworks>
-    <RootNamespace>NServiceBus.NHibernate</RootNamespace>
-    <AssemblyName>NServiceBus.NHibernate</AssemblyName>
+    <TargetFramework>net452</TargetFramework>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <OutputPath>..\..\binaries\</OutputPath>
-    <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
-    <UpdateAssemblyInfo>true</UpdateAssemblyInfo>
-  </PropertyGroup>
-
-  <PropertyGroup>
     <Description>NHibernate integration for NServiceBus</Description>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
   </PropertyGroup>
 
   <ItemGroup>
     <Reference Include="System.Data.DataSetExtensions" />
-    
+  </ItemGroup>
+
+  <ItemGroup>
     <PackageReference Include="Fody" Version="2.*" PrivateAssets="All" />
     <PackageReference Include="Janitor.Fody" Version="1.*" PrivateAssets="All" />
     <PackageReference Include="Obsolete.Fody" Version="4.*" PrivateAssets="All" />
     <PackageReference Include="Particular.CodeRules" Version="0.2.0" PrivateAssets="All" />
+    <PackageReference Include="Particular.Packaging" Version="0.1.0" PrivateAssets="All" />
     <PackageReference Include="NServiceBus" Version="[7.0.0-*, 8.0.0)" />
     <PackageReference Include="NHibernate" version="[4.0.4.4000, 5.0.0)" />
     <PackageReference Include="Iesi.Collections" version="[4.0.2, 5.0.0)" />
```

Comment:
It's not immediately clear to me why this dependency is here, since it's an NHibernate dependency and would be picked up transitively through the NHibernate package reference. Is there something about the 4.0.2 package specifically that we need?

Reply:
I think this is a left-over from before support for transitive dependencies. Thanks for pointing this out!

Reply:
Would you like me to remove it as part of this PR?

Reply:
That would be perfect but I can also remove it in a separate PR later.